.github/workflows/firesim-run-tests.yml

```diff
@@ -269,3 +269,44 @@ jobs:
           python3 -m sphinx.ext.intersphinx docs/_build/html/objects.inv
           cat /tmp/sphinx-err*.log
       - uses: ./.github/actions/job-end
+
+  lints:
+    name: Lints
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/circt/images/circt-ci-build:20210929220136
```

Comment:
I don't think we want to depend on another unrelated project's CI container to run clang-format and not provide the same version to developers to run locally via conda.

If we are going to adopt `clang-format`, we should better integrate this into our existing development environment before merging.

We should also name the step something more descriptive than `lints` unless we're thinking of cramming all of the lint checkers into this one step.  `clang-check` or `cppcheck` are more what I would consider lint checkers, this is just enforcing a formatting standard.

Reply:
I changed this to use a default image. Will look into conda as well.